staff/clara-rufi/skylabcoders-html-semantics/community.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h1>
+            <a href="/en"><img src="/images/403/default.png" alt="Skylab Coders Academy">Skylab Coders Academy</a>
+        </h1>
+        <ul>
+            <li>
+                <a lang="es" href="/es">es</a>
+            </li>
+            <li>
+                <strong>
+                    <a lang="en" href="/en">en</a>
+                </strong>
+            </li>
+            <li>
+                <a href="/ca">ca</a>
+            </li>
+        </ul>
+        <form action="/en/search-results_21631">
+            <input placeholder="" type="search" name="q">
+        </form>
+        <address>
+            <a href="tel:+34931760220">93 176 02 20</a>
+        </address>
+        <button type="button">en</button>
+        <button type="button"><img src="/703.png" alt="SkyLab Coders Academy Menu"></button>
+    </header>
+    <nav>
+        <h1>Main navigation</h1>
+        <ul>
+            <li><strong><a href="https://www.skylabcoders.com/en">Home</a></strong></li>
+            <li>
+                <a href="https://www.skylabcoders.com/en/courses_3574">Courses</a>
+                <ul>
+                    <li>
+                        <a href="https://www.skylabcoders.com/en/full-time-bootcamp_5278">Full time - Bootcamp</a>
+                    </li>
+                    <li>
+                        <a href="https://www.skylabcoders.com/en/part-time-course_25971">
+                            Part time course
+                        </a>
+                    </li>
+                </ul>
+            </li>
+            <li>
+                <a href="https://www.skylabcoders.com/en/faculty_3803">Faculty</a>
+            </li>
+            <li>
+                <a href="https://www.skylabcoders.com/en/why-code-_4139">Why code?</a>
+            </li>
+            <li>
+                <a href="https://www.skylabcoders.com/en/companies_4434">Companies</a>
+            </li>
+            <li>
+                <a href="https://www.skylabcoders.com/en/community_4573">Community</a>
+            </li>
+            <li>
+                <a href="https://www.skylabcoders.com/en/contact_4078">Contact</a>
+            </li>
+        </ul>
+    </nav>
+    <main>
+
+        <section>
+            <h1>Skylab Community</h1>
+
+            <h1>Skylab Community</h1>
+            <p>Skylab Community</p>
+            Skylab Community |
```

Comment:
You haven't put any tag to this text line

Reply:
i solve it, thanks :)